modules/admin_manual/pages/installation/system_requirements.adoc

```diff
@@ -97,21 +79,13 @@ with a MySQL or MariaDB database:
 xref:configuration/database/linux_database_configuration.adoc#mysql-mariadb-storage-engine[MySQL / MariaDB storage engine])
 * `READ COMMITED` transaction isolation level (See: 
 xref:configuration/database/linux_database_configuration.adoc#mysql-mariadb-read-commited-transaction-isolation-level[MySQL / MariaDB `READ COMMITED` transaction isolation level])
-====
-
 
-=== Mobile
```

Comment:
Did we want to lose the "Mobile" section?

Reply:
Correct. I'll add it back in.